gluten-substrait/src/main/scala/org/apache/spark/shuffle/sort/ColumnarShuffleManager.scala

```diff
@@ -107,7 +107,7 @@ class ColumnarShuffleManager(conf: SparkConf) extends ShuffleManager with Loggin
           metrics,
           shuffleExecutorComponents)
       case other: BaseShuffleHandle[K @unchecked, V @unchecked, _] =>
-        new SortShuffleWriter(other, mapId, context, shuffleExecutorComponents)
+        new SortShuffleWriter(other, mapId, context, _, shuffleExecutorComponents)
```

Comment:
Does this mean that gluten will not guarantee spark minor version compatibility?

Reply:
Hi @wForget, 
Not guaranteed for all minor versions indeed. Actually it's the same for Spark 32 & 3.3 & 3.4. It's only tested with the latest minor release. 
The issue is mostly due to the shim layer design, and the CI/CD resources are also not enough for so many small versions.

Reply:
I see, thank you.